rero_mef/concepts/mef/api.py

```diff
@@ -112,6 +114,20 @@ def create_or_update_mef_viaf_record(self, dbcommit=False, reindex=False,
         """
         return self, Action.Error, None, False
 
+    @classmethod
+    def create_deleted(cls, record, dbcommit=False, reindex=False):
+        """Create a deleted record for an record.
+
+        :param record: Record to create.
+        :param dbcommit: Commit changes to DB.
+        :param reindex: Reindex record.
+        :returns: Created record.
+        """
+        data = {record.name: {
+            '$ref': build_ref_string(record.pid, record.name)}}
+        data['deleted'] = datetime.now(timezone.utc).isoformat()
+        return cls.create(data=data, dbcommit=dbcommit, reindex=reindex)
+
```

Comment:
if places into parent `EntityMefRecord`, you can remove due to inheritance.

Reply:
I have to rethink the Classes. Normally I have to create a entity and a MEF class.